makefile

```diff
@@ -0,0 +1,12 @@
+default:
+	cat makefile
+get_synthetic_data:
+	wget -O synthetic_data.csv https://gist.github.com/Niarfe/25100a860cbb660150a3fec87378d8ac/raw/
```

Comment:
ah, nice use of the `-O` flag!  Although I think it would have worked without the flag if you used `https://gist.githubusercontent.com/Niarfe/25100a860cbb660150a3fec87378d8ac/raw/synthetic_data.csv` as the link.  However, kudos to you since there are many times then the file name is unfriendly, like `a00aa3hwbue0w0w0g.001` for example, and a more friendly rename is in order.  Nice 👍